src/Psy/CodeCleaner.php

```diff
@@ -53,7 +52,9 @@ class CodeCleaner
      */
     public function __construct(Parser $parser = null, Printer $printer = null, NodeTraverser $traverser = null)
     {
-        $this->parser    = $parser    ?: new Parser(new Lexer());
+        $parserFactory = new ParserFactory();
```

Comment:
> Hope we will see PHP version increment in the requirements soon...

Probably not, at least until it becomes too arduous to support. PsySH is useful even for people whose legacy support needs preclude them from using PHP 5.4+ :)